src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ContentDbContext.cs

```diff
@@ -29,6 +29,7 @@ public ContentDbContext(DbContextOptions<ContentDbContext> options)
         public DbSet<Topic> Topics { get; set; }
         public DbSet<Publication> Publications { get; set; }
         public DbSet<Release> Releases { get; set; }
+        public DbSet<ReleaseStatus> ReleaseStatus { get; set; }
```

Comment:
All but 4 of our content tables use a plural name which I'm not a big fan of but just letting you know this is inconsistent with the majority of our table names unless it was `ReleaseStatuses`. I only realised this recently when adding the new Methodology tables and not sure I'm a fan of using the plural names but I understand it. I guess we could create a new ticket to standardise the names later on and rename the tables.

Reply:
I've created EES-2434 to address this.